plugins/generator-1.16.5/forge-1.16.5/templates/modbase/variableslist.java.ftl

```diff
@@ -355,16 +355,11 @@ public class ${JavaModName}Variables {
 	}
 
 	@SubscribeEvent public void clonePlayer(PlayerEvent.Clone event) {
-		PlayerVariables original = ((PlayerVariables) event.getOriginal().getCapability(PLAYER_VARIABLES_CAPABILITY, null).orElse(new PlayerVariables()));
-		PlayerVariables clone = ((PlayerVariables) event.getEntity().getCapability(PLAYER_VARIABLES_CAPABILITY, null).orElse(new PlayerVariables()));
-		<#list variables as var>
-			<#if var.getScope().name() == "PLAYER_PERSISTENT">
-			clone.${var.getName()} = original.${var.getName()};
-			</#if>
-		</#list>
-		if(!event.isWasDeath()) {
+		if(event.isWasDeath()) {

```

Comment:
This part should not be altered as these changes do not fall under this PR.

Reply:
Weird... I never changed something at this place. This is the first I see this event. :/

Reply:
Well revert that part so no changes apply here